backend/src/main/resources/data.sql

```diff
@@ -0,0 +1,25 @@
+-- This file contains basic data that is loaded into the database upon startup.
+INSERT INTO skill_type (colour, name) VALUES ('#1EA589', 'Front-end developer');
+INSERT INTO skill_type (colour, name) VALUES ('#B13C38', 'Back-end developer');
+INSERT INTO skill_type (colour, name) VALUES ('#C1FFAD', 'UX / UI designer');
+INSERT INTO skill_type (colour, name) VALUES ('#EC6F1C', 'Graphic designer');
+INSERT INTO skill_type (colour, name) VALUES ('#1C1CEC', 'Business Modeller');
+INSERT INTO skill_type (colour, name) VALUES ('#B88818', 'Storyteller');
+INSERT INTO skill_type (colour, name) VALUES ('#6A2A2D', 'Marketer');
+INSERT INTO skill_type (colour, name) VALUES ('#1CEC1C', 'Copywriter');
+INSERT INTO skill_type (colour, name) VALUES ('#632CB1', 'Video editor');
+INSERT INTO skill_type (colour, name) VALUES ('#A73C97', 'Photographer');
+INSERT INTO skill_type (colour, name) VALUES ('#505369', 'Other');
+
+INSERT INTO communication_template (name, subject, template)
+            VALUES ('Yes', 'Accepted for OSOC',
+                    'We are pleased to tell you that you have been accepted for OSOC!\n Someone will be in touch with you shortly to work out the details.');
+INSERT INTO communication_template (name, subject, template)
+            VALUES ('No', 'Rejected for OSOC',
```

Comment:
Rejected for OSOC sounds a bit harsh as mail subject... Maybe we should change all subjects to OSOC application status to soften the blow?